### ISSUE TITLE
Is there any way to NDEF format NFC tag?

### ISSUE THREAD

**Issue by aventez** [semlette/nfc_in_flutter#84]
My question is how can I NDEF format the empty NFC tag in Flutter? Currently, I'm using this lib, but it only supports already NDEF formatted tags. Any ideas how to solve that?

Here is my code.

```
NDEFMessage newMessage = NDEFMessage.withRecords([
  NDEFRecord.uri(path),
]);

Stream<NDEFTag> stream = NFC.writeNDEF(newMessage, once: true);
stream.listen((NDEFTag tag) {
  showNotification(context, 'NFC Tag was activated.');
});
```

Thanks in advance!

**Comment by ZippyRainbow** [comment]
@aventez I don't believe the function to format a tag to NDEF exists in the code. Would need to be developed as an option.

**Comment by guenth39** [comment]
I think at least for android the necessary code is there. No idea if it works. I do not know how to format a tag differently for testing 😄 

https://github.com/semlette/nfc_in_flutter/blob/8c0550e32d19af82dc482ece65ddb83808003ea4/android/src/main/java/me/andisemler/nfc_in_flutter/NfcInFlutterPlugin.java#L628-L642
